compiler/onecc-docker/README.md

```diff
@@ -0,0 +1,263 @@
+# onecc-docker
+
+`onecc-docker` supports `one-cmd`, which is currently only supported by ubuntu 18.04, in more OS.
+
+
+
+## Description
+
+For now, `one-cmds` tools only support Ubuntu 18.04 and 20.04(not officially).
+So people in other environments can't use our tools unless they upgrade the OS (or install Ubuntu OS).
+
+Therefore, we developed `onecc` that runs using a docker so that users can use `one-cmds` more easily. This is `onecc-docker.`
+
+> Currently, `onecc-docker` is only supported in Linux environments.
+
+
+
+## Requirements
+
+- Linux OS
+
+- Docker
+
+    - `onecc-docker` requires the current `user ID` to be included in the `docker group` because it requires the docker-related commands to be executed without `sudo` privileges.
+
+        ```
+        sudo usermod -aG docker ${USER}
+        ```
+
+- Python 3.8
+
+
+
+## Structure
+
+### Summary 
+
+`onecc-docker` creates a docker image with the latest release version of `ONE`, runs `onecc` inside the docker container after the container is driven to deliver the desired result to the user.
+
+### To check the latest version of ONE
+
+- We use github api to get the latest release version of ONE. Create a docker file using the imported version.
```

Comment:
Maybe `github api` -> `GitHub API`

Reply:
I'm not sure about this line is really necessary.

Reply:
If you mean line 41, I'll remove it. Thank you!

Reply:
No... it's line 42, explanation seems internal implementation(?)
Maybe I couldn't catch the whole story how the explanation is going.
I think this is related with using word `structure`

Reply:
How about using a different name instead of `structure`?
I don't think the title and content match.